utils/credentials/credentials.go

```diff
@@ -43,6 +44,7 @@ var (
 	usernameKey    = "username"
 	passwordKey    = "password"
 	caEnt          *entity.Entity
+	targetName     = "client.com"
```

Comment:
we should make another PR turning this into an exportable flag that can be used by all the binaries

Reply:
Sure